codespell_lib/tests/test_basic.py

```diff
@@ -1081,7 +1081,9 @@ def test_config_toml(
             )
     else:
         assert kind == "toml"
-        pytest.importorskip("tomli")
+        python_version = sys.version_info
+        if python_version.major == 3 and python_version.minor < 11:
```

Comment:
Perhaps just:
```python
        if sys.version_info < (3, 11):
```

Reply:
Thought that might be problematic due to the micro version, but the way python evaluates this your code is simpler and works as well.